src/main/services/win-registry.ts

```diff
@@ -50,10 +52,15 @@ export class WinRegistry {
      */
     public registerWindow(win: BrowserWindow, type: AppWindowType): AppWindow {
         const winId = win.id;
-        const appWindow = {
+        const appWindow: AppWindow = {
             identifier: uuid.v4(),
             type,
             win,
+            setBoundsHandler: () => {
```

Comment:
I think this function should include:
```
win.on("move", setBoundsHandler);
win.on("resize", setBoundsHandler);
```
...to avoid the repetitive calls to the event handler registrations.
`setBoundsHandler(win)` then just becomes a local function in `src/main/services/win-registry.ts` with the `win: BrowserWindow` parameter.

Reply:
...and of course, `removeBoundsHandler()` can be added here too, so that `win.removeListener("move", setBoundsHandler);` and `win.removeListener("resize", readerWindow.setBoundsHandler);` can be called.

Reply:
I made a typo in my comments above, I have now updated them, please discard the email notification and read the code comments directly in GitHub. Thanks! :)

Reply:
`onWindowMoveResize(win)` how do you pass `win` in parameter of event ?

Reply:
`onWindowMoveResize()` can be a local closure where `win` is defined (sorry, I typed my comments in "pseudo code", I didn't go through all the details)

Reply:
done

Reply:
Nice :)